### PR TITLE
Refactor fonts based on V3

### PR DESCRIPTION
**Pull request by NurkaAmre**
## Context

[link to Jira card](https://ywgc.atlassian.net/jira/software/projects/YWGC/boards/1?selectedIssue=YWGC-36)

## Work done
Updated fonts based on Version3
Add new font
## Manual testing instructions
Check all fonts 
Some titles disappear because of bg color apply text-black to check | apply bg
## Gotchas/What I learned
Fonts

### CODE REVIEW

**Review by alliequintano — DISMISSED** [review]
hmm I see what you mean about the font colors. Could we keep the colors as they were for the scope of this PR? This should just adjust the weight and size, and the 'Update ... to new layout' can address the color and placement. Let me know if that doesn't make sense or if you have any questions!

File: tailwind.config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
   theme: {
     extend: {
       fontFamily: {
-        header: ['Fjalla One', 'sans-serif'],
+        number: ['BioRhyme', 'sans-serif'],
```
Comment (jaynaxissa) [inline tailwind.config.js:11]: Are we using BioRhyme? I saw on figma design that it's Monserrat font.

**Review by jaynaxissa — COMMENTED** [review]
Nitpicking(nb): Can we double check what font are we using? Is it BioRhyme or Monserrat? 

Also can we be more specific next time in the manual testing instruction about what we need to check. I know all font needs to be changed but would be easier if it's already mention what are the changes I should be looking at to check. Cheers:)

File: tailwind.config.js (hunk 1 above)
Comment (NurkaAmre) [reply tailwind.config.js:11]: @jaynaxissa we are using BioRhyme font for numbers
image

Comment (jaynaxissa) [reply tailwind.config.js:11]: I see. Can you help us with installing other required fonts too? I think it's included in your PR but correct me if I'm wrong 😃 There's a list on Figma in Desktop/mobile typography section.

Comment (jaynaxissa) [reply tailwind.config.js:11]: Screenshot 2023-08-14 at 14 35 39 (2)
Screenshot 2023-08-14 at 14 35 47 (2)

Are we not using the v3?

Comment (NurkaAmre) [reply tailwind.config.js:11]: @jaynaxissa High-Fidelity-Wireframes its a V3

Comment (NurkaAmre) [reply tailwind.config.js:11]: image

Comment (jaynaxissa) [reply tailwind.config.js:11]: Yes I understand that it's v3 high-fidelity on Figma. If you look on the v3 there's more details about the fonts on there.

Comment (jaynaxissa) [reply tailwind.config.js:11]: At the section called "Desktop Typography"

**Review by ajkarczewski — DISMISSED** [review]
- Button text on desktop should be 20px (tailwind text-xl) bold
- Everything else looks accurate!

**Review by ajkarczewski — COMMENTED** [review]
- It looks like the 20px is not being applied when the screen expands larger -- should be applied anytime the screen is larger than mobile 🤔 
- currently, when i go to mobile size, the button text is 16px (should be 18px)

**Review by novellac — APPROVED** [review]
The imports look fine - please have the designer sign off on the implementation of all the classes.

**Pull request merged**